Userland/Libraries/LibC/grp.h

```diff
@@ -23,6 +24,7 @@ void setgrent();
 void endgrent();
 struct group* getgrnam(const char* name);
 struct group* getgrgid(gid_t);
+int putgrent(const struct group* p, FILE* stream);
```

Comment:
You can and should omit the variable name for arguments if their use is obvious, for example here. A non-obvious use would be a bool argument (unless the function is a setter).